resources/platform/linux/README.md

```diff
@@ -0,0 +1,23 @@
+In order to update the libradamsa shared object, clone the radamsa repo
```

Comment:
I wasn't too clear in the previous comment, but move this and the .so to a subdirectory under this one. It's confusing for this to be included as a README for the full linux directory. That will also need a change to update the path you search for to find the .so.

Reply:
Done